client/dashboard/app/notifications/style.scss

```diff
@@ -1,3 +1,14 @@
+// The Dropdown component doesn't support to hide the header if the `expandOnMobile` is enabled.
+.dashboard-notifications.components-popover.is-expanded {
```

Comment:
Does it make sense to custom styles? Or should we switch to use `Modal` on mobile 🤔

Reply:
Hmm but `Modal` will still have rounded border, not really full screen, no? Will that make it easier to style it 🤔 

<img width="483" height="256" alt="image" src="https://github.com/user-attachments/assets/02c1782d-c488-4a02-a7a6-31f3a862f417" />